Looks like it might be related?
```
======================================================================
FAIL: test_case_template_params (corehq.apps.reminders.tests.test_message_formatting.MessageTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/reminders/tests/test_message_formatting.py", line 143, in test_case_template_params
    self.assertEqual(get_message_template_params(child_case), child_expected_result)
AssertionError: {'case': {'opened_on': '2016-07-19T12:23:08.239584Z', 'domain': u'message-format [truncated]... != {'case': {'opened_on': '2016-07-19T12:23:08.239584Z', 'domain': u'message-format [truncated]...
Diff is 10653 characters long. Set self.maxDiff to None to see it.
```

--- a/corehq/apps/reminders/tests/test_message_formatting.py
+++ b/corehq/apps/reminders/tests/test_message_formatting.py
@@ -106,7 +106,7 @@ class MessageTestCase(TestCase):
 
     def get_expected_template_params_for_mobile(self):
         return {
-            'name': self.mobile_user.username,
+            'name': self.mobile_user.raw_username,
             'first_name': self.mobile_user.first_name,
             'last_name': self.mobile_user.last_name,
         }

